library/core/src/sync/atomic.rs

```diff
@@ -1246,7 +1246,7 @@ impl AtomicBool {
     /// atomic types work with interior mutability. All modifications of an atomic change the value
     /// through a shared reference, and can do so safely as long as they use atomic operations. Any
     /// use of the returned raw pointer requires an `unsafe` block and still has to uphold the same
-    /// restriction: operations on it must be atomic.
+    /// restriction in [Memory model for atomic accesses].
```

Comment:
This doesn't sound like a sentence to me? "restriction in Memory model" is not grammatical, and also oddly capitalized.